content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Crie um Pod de Hostprocess do Windows
+content_type: task
+weight: 20
+min-kubernetes-server-version: 1.23
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+
+Os contêineres de HostProcess do Windows permitem que você execute
+cargas de trabalho em um host Windows contêinerizado. Esses contêineres operam como
+processos normais, mas tem acesso à rede host do namespace,
+armazenamento e dispositivos, quando recebem os privilégios de usuário apropriados.
```

Comment:
Minor: "têm acesso ao namespace, armazenamento e dispositivos da rede host" would be more idiomatic.